questions/models.py

```diff
@@ -18,4 +18,4 @@ class Meta:
         verbose_name_plural = "Types"
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
```

Comment:
How is this relevant to this PR,

Please add commits which is relevant to this PR.